pkg/util/kms/kms_kmip.go

```diff
@@ -0,0 +1,113 @@
+package kms
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// KMIP client config options
+const (
+	KMIPEndpoint      = "KMIP_ENDPOINT"
```

Comment:
no need to export these constants ?

Reply:
Thanks for the review @Rakshith-R 🖖 How to export those?

Reply:
I mean the naming of these variables can me `kmipEndpoint = "KMIP_ENDPOINT"`, 
this would make them local to golang package, since they are not being using anywhere
outside the package in code.